index.html

```diff
@@ -347,23 +347,24 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
         Therefore, this API expects the {{ThingDescription}} objects be validated before used as parameters. This specification defines a basic <a>TD</a> validation as follows.
       </p>
       <div>
-        To <dfn>validate a TD</dfn> given |td:ThingDescription|, run the following steps:
```

Comment:
Same: I don't think algorithm names and params should be removed.

Reply:
I agree, see my review.